.github/workflows/end2end.yaml

```diff
@@ -164,7 +165,7 @@ jobs:
         if: always()
 
   build-doc:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Did you try with `24`?

Reply:
yes, it's documented in the commit message: with 24 we get python 3.12, which breaks the current sphinx... So we need to upgrade all that stuff, too much for today :-)
(but i added a note in the zenko dependency bumps tickets)